CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for Crate Admin Interface
 Unreleased
 ==========
 
+ - Display ``license.ident`` in the Enterprise Edition.
```

Comment:
Also, since this is a thing we're now doing for the next release (but not backporting it), can you add the following sections to the CHANGE.txt under unreleased:

```
Breaking Changes
================

Changes
=======

Fixes
====
```

And to put this in the corresponding section (`Changes` I assume?)